src/Mod/PartDesign/App/FeatureText.cpp

```diff
@@ -0,0 +1,352 @@
+            return new App::DocumentObjectExecReturn("Pad: Resulting shape is empty");
+
+        // set the additive shape property for later usage in e.g. pattern
+        prism = refineShapeIfActive(prism);
+        this->AddSubShape.setValue(prism);
+
+        if (!base.IsNull()) {
+//             auto obj = getDocument()->addObject("Part::Feature", "prism");
+//             static_cast<Part::Feature*>(obj)->Shape.setValue(getSolid(prism));
+            // Let's call algorithm computing a fuse operation:
+            BRepAlgoAPI_Fuse mkFuse(base, prism);
+            // Let's check if the fusion has been successful
+            if (!mkFuse.IsDone())
+                return new App::DocumentObjectExecReturn("Pad: Fusion with base feature failed");
+            TopoDS_Shape result = mkFuse.Shape();
+
+            // we have to get the solids (fuse sometimes creates compounds)
+            TopoDS_Shape solRes = this->getSolid(result);
+            // lets check if the result is a solid
+            if (solRes.IsNull())
+                return new App::DocumentObjectExecReturn("Pad: Resulting shape is not a solid");
+
+            int solidCount = countSolids(result);
+            if (solidCount > 1) {
+                return new App::DocumentObjectExecReturn("Pad: Result has multiple solids. This is not supported at this time.");
+            }
+
+            solRes = refineShapeIfActive(solRes);
+            this->Shape.setValue(getSolid(solRes));
+        } else {
+            int solidCount = countSolids(prism);
+            if (solidCount > 1) {
+                return new App::DocumentObjectExecReturn("Pad: Result has multiple solids. This is not supported at this time.");
+            }
+
+           this->Shape.setValue(getSolid(prism));
+        }
+
+        return App::DocumentObject::StdReturn;
+    }
+    catch (Standard_Failure& e) {
+
+        if (std::string(e.GetMessageString()) == "TopoDS::Face")
+            return new App::DocumentObjectExecReturn("Could not create face from sketch.\n"
+                "Intersecting sketch entities or multiple faces in a sketch are not allowed.");
+        else
+            return new App::DocumentObjectExecReturn(e.GetMessageString());
+    }
+    catch (Base::Exception& e) {
+        return new App::DocumentObjectExecReturn(e.what());
+    }
+}
+
+
+
+void Text::updateAxis(void)
+{
+    App::DocumentObject *pcReferenceAxis = ReferenceAxis.getValue();
+    const std::vector<std::string> &subReferenceAxis = ReferenceAxis.getSubValues();
+    Base::Vector3d base;
+    Base::Vector3d dir;
+    getAxis(pcReferenceAxis, subReferenceAxis, base, dir);
+
+    Base.setValue(base.x,base.y,base.z);
+    Axis.setValue(dir.x,dir.y,dir.z);
+}
+
+
+TopoDS_Shape Text::textToShape(const std::string& text, const std::string& fontpath, double height, double track = 0.0)
+{
+    Base::Console().Warning("textToShape was called\n");
+
+    Py::Module mod(PyImport_ImportModule("Part"), true);
```

Comment:
This line segfaults when recomputing attached file
[example.zip](https://github.com/FreeCAD/FreeCAD/files/5776761/example.zip)